hq/test/schedule/IamUsersToDisableTest.scala

```diff
@@ -0,0 +1,96 @@
+package schedule
+
+import com.gu.anghammarad.models.Stack
+import model._
+import org.joda.time.DateTime
+import org.scalatest.{FreeSpec, Matchers}
+import schedule.IamUsersToDisable.usersToDisable
+
+class IamUsersToDisableTest extends FreeSpec with Matchers {
+
+  private val dynamo: DynamoAlertService = new MockDynamoAlertService()
+
+  private val account = AwsAccount("accountId","accountName","accountRole","accountNumber")
+  def flaggedUserWithUsername(username: String) = Map(
+     account -> Seq(
+      IAMAlertTargetGroup(
+        List(Stack("test-stack")),
+        Seq(
+          VulnerableUser(
+            username = username,
+            humanUser = true,
+            tags = List.empty,
+            disableDeadline = None
+  )))))
+
+  "usersToDisable" - {
```

Comment:
These tests are more complicated than the ones they replace - previously we were only testing one part of the class (an internal method). We could consider if there is any way to simplify the logic that helps with testability, but that felt outside the scope of this PR